src/OrleansCodeGenerator/SerializerGenerationManager.cs

```diff
@@ -117,7 +117,7 @@ internal bool RecordTypeToGenerate(Type t, Module module, Assembly targetAssembl
             return true;
         }
 
-        private static bool IsFieldInaccessibleForSerialization(Module module, Assembly targetAssembly, FieldInfo field)
```

Comment:
It's prep, see the call to `SerializationManager.HasSerialzier`? We know a field can be accessed for serialization if we already have a serializer for it